CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh

```diff
@@ -1,19 +1,19 @@
-conddb --db myfile.db ZDCRECO_Geometry_TagXX ZDCRECO_Geometry_TagXX --destdb ZDCRECO_Geometry.db
-conddb --db myfile.db CASTORRECO_Geometry_TagXX CASTORRECO_Geometry_TagXX --destdb CASTORRECO_Geometry.db
-conddb --db myfile.db CSCRECO_Geometry_TagXX CSCRECO_Geometry_TagXX --destdb CSCRECO_Geometry.db
-conddb --db myfile.db CSCRECODIGI_Geometry_TagXX CSCRECODIGI_Geometry_TagXX --destdb CSCRECODIGI_Geometry.db
-conddb --db myfile.db DTRECO_Geometry_TagXX DTRECO_Geometry_TagXX --destdb DTRECO_Geometry.db
-conddb --db myfile.db RPCRECO_Geometry_TagXX RPCRECO_Geometry_TagXX --destdb RPCRECO_Geometry.db
-conddb --db myfile.db GEMRECO_Geometry_TagXX GEMRECO_Geometry_TagXX --destdb GEMRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t XMLFILE_Geometry_TagXX_Extended2021_mc -i XMLFILE_Geometry_TagXX_Extended2021_mc -c sqlite_file:GeometryFileExtended2021.db
+conddb_import -f sqlite_file:myfile.db -t TKRECO_Geometry_TagXX -i TKRECO_Geometry_TagXX -c sqlite_file:TKRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t TKExtra_Geometry_TagXX -i TKExtra_Geometry_TagXX -c sqlite_file:TKExtra_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t TKParameters_Geometry_TagXX -i TKParameters_Geometry_TagXX -c sqlite_file:TKParameters_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t EBRECO_Geometry_TagXX -i EBRECO_Geometry_TagXX -c sqlite_file:EBRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t EERECO_Geometry_TagXX -i EERECO_Geometry_TagXX -c sqlite_file:EERECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t EPRECO_Geometry_TagXX -i EPRECO_Geometry_TagXX -c sqlite_file:EPRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t HCALRECO_Geometry_TagXX -i HCALRECO_Geometry_TagXX -c sqlite_file:HCALRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t HCALParameters_Geometry_TagXX -i HCALParameters_Geometry_TagXX -c sqlite_file:HCALParameters_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t CTRECO_Geometry_TagXX -i CTRECO_Geometry_TagXX -c sqlite_file:CTRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t ZDCRECO_Geometry_TagXX -i ZDCRECO_Geometry_TagXX -c sqlite_file:ZDCRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t CASTORRECO_Geometry_TagXX -i CASTORRECO_Geometry_TagXX -c sqlite_file:CASTORRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t CSCRECO_Geometry_TagXX -i CSCRECO_Geometry_TagXX -c sqlite_file:CSCRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t CSCRECODIGI_Geometry_TagXX -i CSCRECODIGI_Geometry_TagXX -c sqlite_file:CSCRECODIGI_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t DTRECO_Geometry_TagXX -i DTRECO_Geometry_TagXX -c sqlite_file:DTRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t RPCRECO_Geometry_TagXX -i RPCRECO_Geometry_TagXX -c sqlite_file:RPCRECO_Geometry.db
+conddb_import -f sqlite_file:myfile.db -t GEMRECO_Geometry_TagXX -i GEMRECO_Geometry_TagXX -c sqlite_file:GEMRECO_Geometry.db
```

Comment:
Please revert all the conddb_import based command to conddb. We are aiming to remove completely conddb_import

Reply:
@ggovi The previous version does NOT run! Can we just merge this PR, and then DB can make whatever changes they want?

Reply:
What is the 'previous version'? What do you mean does not run?

Reply:
conddb_import -f sqlite_file:myfile.db -t TKRECO_Geometry_TagXX -i TKRECO_Geometry_TagXX -c sqlite_file:TKRECO_Geometry.db 
is equivalent to:
conddb --db myfile.db copy TKRECO_Geometry_TagXX --destdb TKRECO_Geometry.db

Reply:
> What is the 'previous version'? What do you mean does not run?
@ggovi - this is how the `conddb` command has been recommended:
https://github.com/cms-sw/cmssw/pull/27693#pullrequestreview-273704957
```zsh
conddb --db [input file] [input_tag] [destination_tag] --destdb [dest_file]
```
Here is the error:
```
conddb --db myfile.db dump TKRECO_Geometry_TagXX TKRECO_Geometry_TagXX --destdb TKRECO_Geometry.db
usage: conddb [-h] [--db DB] [--verbose] [--quiet] [--yes] [--nocolors]
              [--editor EDITOR] [--force] [--noLimit] [--authPath AUTHPATH]
              {help,init,status,list,listTags,listGTsForTag,listGTs,listRuns,diff,search,copy,edit,delete,dump,showFCSR,toLumi,fromLumi,toTimestamp,fromTimestamp}
              ...
conddb: error: unrecognized arguments: TKRECO_Geometry_TagXX --destdb TKRECO_Geometry.db
```
where:
```
conddb --db myfile.db listTags
[2020-03-17 17:43:22,419] INFO: Connecting to myfile.db [sqlite:///myfile.db]
Name                             TimeType  ObjectType             Synchronisation  EndOfValidity  Insertion_time              Description   
-------------------------------  --------  ---------------------  ---------------  -------------  --------------------------  -----------   
XMLFILE_Geometry_Extended_TagXX  Run       FileBlob               any              -1             2020-03-06 01:31:20.811129  New Tag       
TKRECO_Geometry_TagXX            Run       PGeometricDet          any              -1             2020-03-06 01:31:23.594264  New Tag       
TKExtra_Geometry_TagXX           Run       PGeometricDetExtra     any              -1             2020-03-06 01:31:23.751784  New Tag       
TKParameters_Geometry_TagXX      Run       PTrackerParameters     any              -1             2020-03-06 01:31:23.753814  New Tag       
EBRECO_Geometry_TagXX            Run       PCaloGeometry          any              -1             2020-03-06 01:31:25.205919  New Tag       
EERECO_Geometry_TagXX            Run       PCaloGeometry          any              -1             2020-03-06 01:31:25.965568  New Tag       
EPRECO_Geometry_TagXX            Run       PCaloGeometry          any              -1             2020-03-06 01:31:30.456739  New Tag       
HCALRECO_Geometry_TagXX          Run       PCaloGeometry          any              -1             2020-03-06 01:31:31.497113  New Tag       
CTRECO_Geometry_TagXX            Run       PCaloGeometry          any              -1             2020-03-06 01:31:31.609155  New Tag       
ZDCRECO_Geometry_TagXX           Run       PCaloGeometry          any              -1             2020-03-06 01:31:31.621453  New Tag       
CASTORRECO_Geometry_TagXX        Run       PCaloGeometry          any              -1             2020-03-06 01:31:31.623177  New Tag       
CSCRECO_Geometry_TagXX           Run       RecoIdealGeometry      any              -1             2020-03-06 01:31:31.951899  New Tag       
CSCRECODIGI_Geometry_TagXX       Run       CSCRecoDigiParameters  any              -1             2020-03-06 01:31:31.952849  New Tag       
DTRECO_Geometry_TagXX            Run       RecoIdealGeometry      any              -1             2020-03-06 01:31:32.436588  New Tag       
RPCRECO_Geometry_TagXX           Run       RecoIdealGeometry      any              -1             2020-03-06 01:31:32.705076  New Tag       
 
```

Reply:
Actually, there is a `copy` command missing:
```
conddb --db myfile.db copy TKRECO_Geometry_TagXX TKRECO_Geometry_TagXX --destdb TKRECO_Geometry.db
[2020-03-17 17:46:52,095] INFO: Connecting to myfile.db [sqlite:///myfile.db]
[2020-03-17 17:46:52,160] INFO: Connecting to TKRECO_Geometry.db [sqlite:///TKRECO_Geometry.db]
[2020-03-17 17:46:52,163] INFO: Initializing database...
[2020-03-17 17:46:52,250] INFO: Copying tag myfile.db::TKRECO_Geometry_TagXX to TKRECO_Geometry.db::TKRECO_Geometry_TagXX from since=1
[2020-03-17 17:46:52,266] INFO: Selected 1 source iov(s)
[2020-03-17 17:46:52,266] INFO: Destination tag synchronization is any
[2020-03-17 17:46:52,266] INFO: Found 1 iovs and 1 referenced payloads to copy.
[2020-03-17 17:46:52,272] INFO: Copying payload myfile.db::036320e37988b24500d6222925d18823318bb31b to TKRECO_Geometry.db::036320e37988b24500d6222925d18823318bb31b ...
[2020-03-17 17:46:52,314] INFO: 1 payload(s) copied.
[2020-03-17 17:46:52,388] INFO: 1 iov(s) copied.
Confirm changes? [n] y

```

```
conddb --db TKRECO_Geometry.db  listTags
[2020-03-17 17:48:30,631] INFO: Connecting to TKRECO_Geometry.db [sqlite:///TKRECO_Geometry.db]
Name                   TimeType  ObjectType     Synchronisation  EndOfValidity  Insertion_time              Description   
---------------------  --------  -------------  ---------------  -------------  --------------------------  -----------   
TKRECO_Geometry_TagXX  Run       PGeometricDet  any              -1             2020-03-17 16:46:52.239678  New Tag       
 
```

Reply:
@ianna @cvuosalo 
as you can read from your conddb usage dump, the line:
conddb --db [input file] [input_tag] [destination_tag] --destdb [dest_file]
does mean nothing. You are missing a command like 'list' or 'copy' , they are listed within brackets.

Reply:
Ok, my suggestion contained a mistake but you can ( should ) test the command before the PR. And if does not work, and the help info is not enough, you can still ask support...

Reply:
@ggovi I would like to make an immediate follow-up pull request to this one to update this file to use `conddb` and also to add an enhancement to `runDDDvsDBGeometryValidation.sh`.
Could you please approve this PR so it can be merged? Then I will make a new PR. That will be easier and more efficient for me. Thanks.